features/update.yml

```diff
@@ -0,0 +1,6 @@
+name: Update frequency media query
+description: The `update` CSS media query sets styles based on whether and how fast the user's device can modify display after it has been rendered.
```

Comment:
Can we say something here about animations? I feel like that's the main thing this is going to be used for: avoiding animations on devices that can't/won't show them smoothly.